onmt/translate/translator.py

```diff
@@ -988,16 +991,6 @@ def _align_forward(self, batch, predictions):
 
     def translate_batch(self, batch, attn_debug):
         """Translate a batch of sentences."""
-        batch_size = len(batch["srclen"])
```

Comment:
you need to keep this :)

Reply:
It is not used

Reply:
you're right thought it was used just underneath